README.md

```diff
@@ -659,10 +659,290 @@ To begin coding the basic execution functionality of our VM, we only need these
 
 #### Compute operands
 
+Once the instruction has been decoded, it is executed by `run_instruction` whose first function is to compute operands. This function is in charge of
+calculating the addresses of the operands and fetching them from memory. If the function could not fetch them from the addresses then they are deduced from the other operands and
```

Comment:
I see, now I get what `them` is in `could not fetch them`. Maybe we can change this to:

```suggestion
calculating the addresses of the operands and fetching them from memory. If the function could not fetch the operands then they are deduced from the other operands and
```